cluster-autoscaler/core/utils/pod_schedulable.go

```diff
@@ -83,3 +83,4 @@ func (podMap PodSchedulableMap) Set(pod *apiv1.Pod, err *simulator.PredicateErro
 		schedulingError: err,
 	})
 }
+
```

Comment:
nit: remove emtpy line

Reply:
It was only in intermediate commit. Removed anyway.